FLEDGE.md

```diff
@@ -151,12 +151,17 @@ const myAuctionConfig = {
                         'https://www.another-buyer.com': 200,
```

Comment:
Can we delete the "additionalBids" line (line 144) above as it's superseded by this PR?  Likewise with the "additionalBids" paragraph below (new line 179)?

Reply:
Done.